catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -43,12 +48,20 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
 
     Map<String, String> aliases =
         (Map<String, String>) arguments.getOrDefault("aliases", new HashMap<>());
-
+    String attributeString =
+        arguments.get("columnOrder") != null ? (String) arguments.get("columnOrder") : "";
```

Comment:
columnOrder is defined in a constant here
https://github.com/codice/ddf/blob/b054ac205251e31390e5ccc9830dfe310a3f2659/catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvQueryResponseTransformer.java#L46